assets/app/views/directives/osc-key-values.html

```diff
@@ -34,38 +65,82 @@
             </span>
           </span>
         </div>      
+      
       </form>
-      <ul class="list-unstyled label-list">
-        <li ng-repeat="(key,value) in entries | valuesIn:readonlyKeys">
-          <span class="key truncate">{{key}}</span>
-          <span class="value truncate">{{value}}</span>
-        </li>
-        <li ng-repeat="(key,value) in entries | valuesNotIn:readonlyKeys">
-          <span ng-controller="KeyValuesEntryController">
-            <span class="key truncate">{{key}}</span>
-            <span ng-hide="editing">
-              <span class="value truncate">{{ value}}</span>
-              <a href="" ng-click="edit()" class="btn btn-default btn-xs" title="Edit">
+      
+      <div style="margin-top: 20px">
```

Comment:
Does `class="gutter-top"` look much different? If it's close, we might go with that so we're using consistent margins as other places.